index.js

```diff
@@ -58,6 +58,31 @@ function onInitHooks() {
 	)
 }
 
+function onInitCreateAdmin() {
+	const email = process.env.BALENA_ROOT_EMAIL || '';
+	const password = process.env.BALENA_ROOT_PASSWORD || '';
+
+	if (email == '' || password == '') {
+		return;
+	}
+
+	console.log("Creating admin account...")
```

Comment:
This should be single quotes, did prettier not run?

Reply:
It did... weird, I shall take a look...

Reply:
It did run, so I don't know what happened here...